packages/base/src/config/CalendarType.ts

```diff
@@ -17,7 +17,28 @@ const getCalendarType = (): CalendarType => {
 		return calendarType;
 	}
 
-	return CalendarType.Gregorian;
+	const legacyDateFormat = getFormatSettings().legacyDateFormat;
```

Comment:
I think you can directly import "getLegacyDateFormat" and use it here (exported from confog/FormatSettings.ts)